src/MultiFormGoals/resources/js/components/color-control/index.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * Vendor dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+const { useInstanceId } = wp.compose;
+const { BaseControl, ColorPalette } = wp.components;
+const { __ } = wp.i18n;
+
+const ImageControl = ( { name, label, help, className, value, hideLabelFromVision, onChange } ) => {
```

Comment:
The variable name is incorrect.

Reply:
@henryholtgeerts can you get this fixed and then merge?

Reply:
Ah! Nice catch, done: https://github.com/impress-org/givewp/pull/5307/commits/da0612a7f53bf2256b52642f67f3c1375b2427c4